src/CustomComponents/NavBar.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {FaBars} from 'react-icons/fa'
+import './NavBar.css';
+import Button1 from "./GenericButton/Button1";
+
+class NavBar extends React.Component{
+    render(){
+      
+        return(
+        <div>
+        <nav>
+            <ul>
+                <li class="logo">CryptoBara</li>
+                <li class="items"><a href="#">Home</a></li>
```

Comment:
Can we turn these href # into props. That way we can pass in the links when we create the component. That way we can reuse this navbar if we need to for other situations.

Reply:
This can be done in the next sprint, with the routers. Mapping out the pages is the next step. The "#" were there for fillers